src/app/components/card/card.component.ts

```diff
@@ -12,7 +12,8 @@ export class CardValue {
   hasLink: Boolean;
   avatar?: string;
   link?: string;
-  menuItem?: any;
+  menuItem?: object;
```

Comment:
This has been resolved here https://github.com/fabric8-ui/fabric8-planner/pull/2068/commits/ebdc10199168c4454e1342df77a706770a442aef